packages/calling/package.json

```diff
@@ -44,7 +44,6 @@
     "jest-html-reporters": "3.0.11",
     "platform": "1.3.6",
     "uuid": "8.3.2",
-    "webex": "1.154.1",
```

Comment:
Did we remove this intentionally?

Reply:
Yes @rarajes2 . We removed this because we don't use `webex` as a dependency this way. We actually never used it even while this was in the internal repo. This should have been removed long ago.